rust/agents/relayer/src/msg/metadata/aggregation.rs

```diff
@@ -107,7 +107,14 @@ impl AggregationIsmMetadataBuilder {
 
         let metas_and_gas_count = metas_and_gas.len();
         if metas_and_gas_count < threshold {
-            info!("Could not fetch all metadata: Found {metas_and_gas_count} of the {threshold} required ISM metadata pieces");
+            info!("Could not fetch all metadata: Found {metas_and_gas_count} of the {threshold} required ISM metadata pieces for message_id {message_id}", metas_and_gas_count=metas_and_gas_count, threshold=threshold, message_id=message.id());
+            for (module_type, ism_address) in invalid_isms {
```

Comment:
I'd rather print all of these as part of the `info` log on the line above, since all are related to the same error. You can just add `?invalid_isms` to the list of args in `info`